src/transformation/to_trimesh/cuboid_to_trimesh.rs

```diff
@@ -28,36 +37,54 @@ impl Cuboid {
  *
  * The cuboid is centered at the origin, and has its half extents set to 0.5.
  */
-fn unit_cuboid() -> (Vec<Point3<Real>>, Vec<[u32; 3]>) {
-    let mut coords = Vec::with_capacity(8);
-    let mut faces = Vec::with_capacity(12);
+fn unit_cuboid() -> (Vec<PointDim<Real>>, Vec<[u32; 3]>) {
+    #[cfg(feature = "dim3")]
+    return {
```

Comment:
Not 100% sure, but, I don’t think you need the `return`? (Assuming the `;` after `}` is removed.)

Reply:
yes that works :+1: